SConstruct

```diff
@@ -131,14 +128,13 @@ opts.Add(BoolVariable("werror", "Treat compiler warnings as errors", False))
 opts.Add(BoolVariable("dev", "If yes, alias for verbose=yes warnings=extra werror=yes", False))
 opts.Add("extra_suffix", "Custom extra suffix added to the base filename of all generated binary files", "")
 opts.Add(BoolVariable("vsproj", "Generate a Visual Studio solution", False))
-opts.Add(EnumVariable("macports_clang", "Build using Clang from MacPorts", "no", ("no", "5.0", "devel")))
```

Comment:
I moved this to `platform/osx/detect.py`. I *think* it should still work but I haven't tested.

The same can't be done easily for `vsproj` which is used in `SConstruct` and assumed to be in `env` (could be changed but that's a more significant refactoring).